addon-resizer/nanny/estimator.go

```diff
@@ -56,3 +56,18 @@ func (e LinearEstimator) scaleWithNodes(numNodes uint64) *api.ResourceRequiremen
 		Requests: requests,
 	}
 }
+
+// LogarithmicEstimator estimates the amount of resources in the way that
+// prevents from frequent updates but may end up with larger resource usage
+// than actually needed (though no more than 50%).
+type LogarithmicEstimator struct {
+	linearEst LinearEstimator
+}
+
+func (e LogarithmicEstimator) scaleWithNodes(numNodes uint64) *api.ResourceRequirements {
+	n := uint64(16)
```

Comment:
The object is very much a mathematical construct. From the object's POV this should be parameterized (even if it's just left as a default for the program).